MIGRATING-0.6-0.7.md

```diff
@@ -281,10 +435,9 @@ For example, a styled rectangle is now built like this:
 
 ## Primitives
 
-### Styling
+Previously, drawing a filled shape with a transparent stroke of non-zero width would bleed the fill under the stroke. This is changed in 0.7 to honor the stroke color, even if it is `None`, allowing for filled shapes with transparent borders.
 
-TODO: Drawing primitives with transparent strokes will now affect the fill area.
-TODO: The behavior of the `Dimensions` impls has changed
+A stroke or fill color of `None` now does not affect the primitive's dimensions.
```

Comment:
IMO we should emphasize that the "dimensions" are the return value of `Dimensions::bounding_box`:
```suggestion
The color of stroke and fill no longer affects the primitive's bounding, that is returned by `Dimensions::bounding_box`. The stroke width is now always considered even if the stroke is transparent.
```

Reply:
Thank you! I was struggling to find good wording for this behaviour.